intercom-rails.gemspec

```diff
@@ -19,14 +19,13 @@ Gem::Specification.new do |s|
   s.test_files = Dir["test/**/*"]
 
   s.add_dependency 'activesupport', '>4.0'
+
   s.add_development_dependency 'rake'
   s.add_development_dependency 'actionpack', '>5.0'
   s.add_development_dependency 'rspec', '~> 3.13'
   s.add_development_dependency 'rspec-rails', '~> 5.0'
   s.add_development_dependency 'pry'
-  s.add_development_dependency 'sinatra', '~> 2.0'
-  s.add_development_dependency 'thin', '~> 1.7.0'
-  s.add_development_dependency 'bigdecimal', '1.3.5'
```

Comment:
Not needed? I couldn't install these on mac silicon. I was going to bump them but removing them appears to have done nothing. 

I think big decimal might be included in modern versions of ruby?

Reply:
Look like 1.3.5 version is for ruby version before 2.5 if you want to `use BigDecimal.new and subclassing without warning`, see [here](https://github.com/ruby/bigdecimal)
<img width="901" alt="Screenshot 2024-12-19 at 14 38 18" src="https://github.com/user-attachments/assets/d7921b60-22ba-490a-b972-28ed0ec5a60d" />

But I'm not sure if we actually need it in our gem.